src/poliastro/tests/test_czml.py

```diff
@@ -67,6 +67,38 @@ def test_czml_add_orbit():
     assert extractor.czml[1]["path"]["show"]["boolean"] is False
 
 
+def test_czml_ground_station():
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    sample_points = 10
+
+    extractor = CZMLExtractor(start_epoch, end_epoch, sample_points)
+
+    extractor.add_ground_station(
+        [1216469.9, -4736121.7, 4081386.9],
+        id_name="GS",
+        label_fill_color=[120, 120, 120, 255],
+        label_text="GS test",
+    )
+
+    extractor.add_ground_station([0.70930, 0.40046], label_show=False)
+
+    assert extractor.gs_czml[0]["id"] == "GS0"
```

Comment:
``czml`` and ``gs_czml`` are kinda interchangeable. I believed it was better to define both separately as this made easier to determine when ground stations and orbital objects where defined. ``extract()`` is more or less a wrapper around ``json.dumps`` (since a python dict isn't a valid json object)

Reply:
Even if we make testing a bit more difficult, I propose we don't separate `czml` and `gs_czml`. What about having only one `export()` method returning a Python dict, so users can decide whether to use `json.dumps` or `json.dump`?

Reply:
As in ``export()`` being a getter of ``czml`` and setting ``czml`` as private (well, as far as Python allows such things)? I don't disagree but wouldn't it be more convenient to still allow ``export()`` to dump ``czml`` directly to a file?